tests/unit/sagemaker/test_serializers.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2017-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import json
+import os
+
+import numpy as np
+import pytest
+
+from sagemaker.serializers import JSONSerializer
+from tests.unit import DATA_DIR
+
+
+@pytest.fixture
+def json_serializer():
+    return JSONSerializer()
+
+
+def test_json_serializer_numpy_valid(json_serializer):
+    result = json_serializer.serialize(np.array([1, 2, 3]))
+
+    assert result == "[1, 2, 3]"
```

Comment:
I think I didn't really notice this before because I know you're just copying the tests over, but I believe the expected result is actually supposed to go first in the comparison. (The order is for the error message when the assertion fails.)